Test/com/nicoles_change_maker/test/ChangeMakerTest.java

```diff
@@ -0,0 +1,235 @@
+package com.nicoles_change_maker.test;
+import com.nicoles_change_maker.ChangeMaker;
+import org.junit.*;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+
+/**
```

Comment:
Here's a classic example of a garbage IDE comment. Everything included in this comment is already there in Git. The quickest way to remove these is to put your cursor over these lines in IntelliJ, hit Option + Enter, and choose "Edit Template." Then delete everything in the template text area and save. It should automatically remove all of these.